templates/pages/brand.html

```diff
@@ -21,6 +21,7 @@
     </div>
 {{/if}}
 <h1 class="page-heading">{{brand.name}}</h1>
+{{{region name="brand_below_header"}}}
```

Comment:
Can you please explain to me the reasoning behind when to use `bottom` and when to use `below` ?

Reply:
Hey Juned, similar to what Erik was stating above, but I believe we opted for `below` / 'above' as it would specify more so where a region would be.